TanukiKit/User.swift

```diff
@@ -3,20 +3,60 @@ import RequestKit
 
 // MARK: model
 @objc public class User: NSObject {
+    public var name: String?
+    public var username: String?
     public let id: Int
-    public var login: String?
+    public var state: String?
+    public var avatarURL: String?
+    public var webURL: String?
+    public var createdAt: String?
+    public var isAdmin: Bool?
     public var bio: String?
-    public var name: String?
+    public var location: String?
+    public var skype: String?
+    public var linkedin: String?
+    public var twitter: String?
+    public var websiteURL: String?
+    public var lastSignInAt: String?
+    public var confirmedAt: String?
     public var email: String?
+    public var themeId: Int?
+    public var colorSchemeId: Int?
+    public var projectsLimit: Int?
+    public var currentSignInAt: String?
+    public var canCreateGroup: Bool?
+    public var canCreateProject: Bool?
+    public var twoFactorEnabled: Bool?
+    public var external: Bool?
     public var privateToken: String?
 
     public init(_ json: [String: AnyObject]) {
         if let id = json["id"] as? Int {
+            name = json["name"] as? String
+            username = json["username"] as? String
             self.id = id
-            login = json["username"] as? String
```

Comment:
This property was changed from `login` to `name` and is a breaking change in the API, I would rather keep it without being a breaking change for now. Can you rename it to `login`, please?